### PR TITLE
Add January 2020 Minutes

### PR DESCRIPTION
**Pull request by matopher**
This adds the board minutes for the meeting on January 21, 2020.

### CODE REVIEW

**Comment by robolucas** [comment]
Hey new Board! Can you remove me from this distribution list?

Thanks!

Lucas

via mobile

**Comment by kacollins** [comment]
@robolucas Here's how you can "unwatch" a repo: https://help.github.com/en/github/receiving-notifications-about-activity-on-github/watching-and-unwatching-repositories#unwatching-a-single-repository

File: 2020/01_january_minutes.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+Approved short-term event planner for networking event, will pay for itself. Model we’d like to use in other areas.
+Alcohol policy needs to be updated, was approved to go against
+
+## Board Reports
+
+Everyone on board to donate at least \$1 by Feb 1.
+
+### Thank you callouts
+
+Ryan Bopp – leading networking event, doing an excellent job
```
Comment (kacollins) [inline 2020/01_january_minutes.md]: Should be "Bupp"

File: 2020/01_january_minutes.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+Bree Weathers – Techlahoma community member, graduate of ctrl shift school, helping out with alcohol policy and handling alcohol at networking event
```
Comment (kacollins) [inline 2020/01_january_minutes.md]: Should be "Bri"

**Review by kacollins — APPROVED** [review]
Thanks!

**Pull request merged**